composer.json

```diff
@@ -76,23 +76,28 @@
     "require": {
         "drupal/allowed_formats": "^1.2",
         "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+        "drupal/config_pages": "^2.7",
         "drupal/ctools": "^3.4",
         "drupal/default_content": "^1.0-alpha8",
+        "drupal/field_encrypt": "^2.0@alpha",
         "drupal/field_group": "^3.0",
         "drupal/layout_builder_restrictions": "^2.7",
         "drupal/menu_block": "^1.6",
         "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
         "drupal/paragraphs": "^1.11",
         "drupal/pathauto": "^1.8",
+        "drupal/real_aes": "^2.3",
         "drupal/shs": "^1.0@alpha",
         "drupal/taxonomy_menu": "dev-3.x",
         "drupal/view_unpublished": "^1.0@alpha",
         "drupal/views_block_filter_block": "^1.0",
         "drupal/views_contextual_filters_or": "^1.1",
         "drupal/views_infinite_scroll": "^1.6",
         "drupal/views_taxonomy_term_name_depth": "^6.0",
-        "su-sws/jumpstart_ui": "^8.1.5",
-        "su-sws/stanford_text_editor": "^8.1.6"
+        "sainsburys/guzzle-oauth2-plugin": "^3.0",
+        "su-sws/jumpstart_ui": "dev-8.x-1.x",
+        "su-sws/stanford_migrate": "dev-8.x-1.x",
+        "su-sws/stanford_text_editor": "dev-8.x-1.x"
```

Comment:
@imonroe shouldn't these be tags?

Reply:
They should indeed.  Right now, my scripts don't check the composer.json's for dev versions, but it's worth adding some additional logic for just this sort of thing.  I'll make a ticket.